docs/development/building-from-sources.md

```diff
@@ -96,7 +96,8 @@ they are case sensitive.
 
 If `PYODIDE_PACKAGES` is not set, a minimal set of packages necessairy to run
 the core test suite is installed, including "micropip", "pyparsing", "pytz",
-"packaging", "Jinja2". This is equivalent to setting `PYODIDE_PACKAGES='core'`
+"packaging", "Jinja2", "regex". This is equivalent to setting
```

Comment:
I had to add it for testing, as none of the other had vendored tests. It's good it have a package that needs compilation in the core meta-package in any case though.